### PR TITLE
Add arguments page

### PR DESCRIPTION
**Pull request by csauve**
Adds a page listing Halo's command line arguments and providing basic instructions to Windows users on how to edit a shortcut. This required adding some basic CSS for tables.

tmp-pr

### CODE REVIEW

File: src/assets/style.scss [hunk 1]
```diff
@@ -104,6 +105,18 @@ h1, h2, h3, h4, h5, h6 {
...
 }
 
+table {
+  border-collapse: collapse;
+  td, th {
+    padding: $padding-containers-small;
+    border: 1px solid $border-color;
+  }
+  th {
+    text-align: left;
```
Comment (csauve) [inline src/assets/style.scss:115]: Alignment can still be overriden from markdown with the `:` symbol:

```md
| header    | header 2
|----------:|---------
| right     | left
```

File: src/content/games/h1/arguments/readme.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+<a href="windows-shortcut.jpg">
+  <img src="windows-shortcut.jpg" alt="Dialog box showing how to add arguments to a Windows shortcut" style="max-width:300px"/>
+</a>
+
+# Arguments list
+
+| Argument           | Description
+|--------------------|----------------
+|`-?`                | Displays a list of all arguments.
+|`-init example.txt` | Causes the game or dedicated server to run the file of console commands on startup.
```
Comment (gbMichelle) [inline src/content/games/h1/arguments/readme.md:20]: Should say `-exec` instead of init

File: src/content/games/h1/arguments/readme.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+|`-ip x.x.x.x`       | Server IP address used when you have multiple IP addresses (e.g. multiple network interfaces in your machine). Behind [NAT][about-nat], this should refer to the [private IP][about-ips] configured for port forwarding.
+|`-screenshot`       | Enables the "Print Screen" key to generate [TGA format][about-tga] screenshots in Halo's `screenshots` directory.
+|`-timedemo`         | A benchmarking mode which runs four movies and writes out `timedemo.txt`. Crashes in Custom Edition.
+|`-devmode`          | Enables developer mode commands ([Custom Edition][h1] only). In pre-1.10 versions, this also prevented Internet gameplay. Note that most client mods like HAC2 and Chimera enable this automatically.
+|`-console`          | Enables the debugging console, which can be opened with the `~` (tilde) key. This console can be used to enter Halo script commands and is similar to Sapien's console, though many Sapien-related commands have no effect. Note that most client mods like HAC2 and Chimera enable this automatically.
```
Comment (gbMichelle) [inline src/content/games/h1/arguments/readme.md:41]:
```suggestion
|`-console`          | Enables the debugging console, which can be opened with the `~` (tilde) key. This console can be used to enter Halo script commands and is similar to Sapien's console, though many Sapien-related commands have no effect. Note that most client mods like HAC2 and Chimera enable this automatically.
|`-name`             | Sets the name of the player profile
|`-path`             | Sets the path for which Halo looks for profiles and gametypes. Normally `%USERPROFILE%\Documents\My Games\Halo[ CE]`
|`-usefxfile`        | Loads shaders from fx files. Does not work.
|`-testcrash`        | Bring up crash dialog.
```

There is also `-use2a`

File: src/content/games/h1/arguments/readme.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+<a href="windows-shortcut.jpg">
+  <img src="windows-shortcut.jpg" alt="Dialog box showing how to add arguments to a Windows shortcut" style="max-width:300px"/>
+</a>
+
+# Arguments list
+
+| Argument           | Description
+|--------------------|----------------
+|`-?`                | Displays a list of all arguments.
```
Comment (gbMichelle) [inline src/content/games/h1/arguments/readme.md:19]:
```suggestion
|`-?`                | Displays a list of most arguments.
|`-help`             | Same as `-?`.

File: src/content/games/h1/arguments/readme.md (hunk 3 above)
Comment (gbMichelle) [reply src/content/games/h1/arguments/readme.md:41]: There is also `-refresh` which likely sets the refresh rate.

**Pull request merged**